cocos2d/core/CCNode.js

```diff
@@ -3094,29 +3094,57 @@ var Node = cc.Class({
 });
 
 /**
+ * !#en
+ * The position changing event, you can listen this event through the statement cc.node.on(cc.Node.EventType.POSITION_CHANGED, callback, this);
```

Comment:
The position changing event, you can listen this event through the statement 

->

The position changing event, you can listen to this event through the statement